test/unit/specs/events/cd-event-add-child-tickets.spec.js

```diff
@@ -229,7 +166,7 @@ describe('Add Child Ticket', () => {
         vm.tickets = [{
           approvedApplications: 0,
           deleted: 0,
-          entity$: 'ticketentity',
+          entity: 'ticketentity',
```

Comment:
err. By remove, i meant, remove the entity$ field alltogether, not just the $ :p